Formula/coredns.rb

```diff
@@ -0,0 +1,23 @@
+class Coredns < Formula
+  desc "Plugin-driven DNS and service discovery"
+  homepage "https://coredns.io"
+  url "https://github.com/coredns/coredns/archive/v1.3.1.tar.gz"
+  sha256 "b471e26c40140fdb05039b36cb8fb804b03ceb329d48a592cbca9ad6550c8143"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/coredns/coredns").install buildpath.children
+
+    cd "src/github.com/coredns/coredns" do
+      system "make", "godeps", "all"
+      bin.install "coredns"
+    end
+  end
+
+  test do
+    "#{bin}/coredns -dns.port=1053"
```

Comment:
- To run a system command you need to call `system`.
- We need to check that it works fine.

Reply:
I haven't been able to test yet because I keep getting `Testing requires the latest version of coredns` because Homebrew is incorrectly inferring that the most recent version is `64` (based on the `64` in the URL). Once I get that figured out I'll be able to write a better test.